scripts/start_components.py

```
@@ -246,7 +303,31 @@ def createNetwork():
+            port= "3306",
+            autocommit=True 
+        )
+        if conn.is_connected():
+            cursor = conn.cursor()
+            cursor.execute("SELECT 1;")
+            cursor.fetchone()
+            conn.close()
+            print("Database is live.")
+            return True
+        
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)